app/components/viral/dialog/header_component.html.erb

```diff
@@ -1,7 +1,7 @@
 <%= render Viral::BaseComponent.new(tag: "div", **system_arguments) do %>
-  <h3 class="dialog--title">
+  <h1 class="dialog--title">
```

Comment:
**nitpick:** Why did you change this to an h1 tag? There should only ever be 1 h1 tag on a page, and the title of a dialogue is probably not that.  The flowbite documentation has it as an h3.

Reply:
Apparently, you are correct.  Thanks for updating this.